app/assets/stylesheets/govuk-component/govspeak/_example.scss

```diff
@@ -7,9 +7,16 @@
 
 .govuk-govspeak {
   .example {
-    border-left: 10px solid $panel-colour;
-    padding-left: 1.5em;
-    margin: 2.5em 0;
+    border-left: 1em solid $panel-colour;
+    padding: 1em 0 1em 1em;
```

Comment:
I know it's a big change that's why I'm wondering why we'd change it only here?

Or is this something we don't keep to standard for on GOV.UK Publishing?